{
  "AppSettings": {
    "EPPIApiUrl": "http://localhost/eridentityprovider",
    "EPPIApiClientSecret": "KlasifikiClientSecret4Devs",
    "EPPIApiClientName": "***REMOVED***",
    "DataServiceDB": "Server=localhost;Database=DataService;Integrated Security=True;",
    "ER4DB": "Server=localhost;Database=Reviewer;Integrated Security=True;",
    "ER4AdminDB": "Server=localhost;Database=ReviewerAdmin;Integrated Security=True;",
    "CochraneOAuthSS": "***REMOVED***", //this works for the test environment.
    //Real string is in [...group folders...]\ER4\Cochrane\oAuth\EPPI-Reviewerd.txt
    "CochraneoAuthBaseAddress": "https://test-login.cochrane.org/",
    //value below needs to match a known CallbackURI within Archie!
    //production value should be: "https://eppi.ioe.ac.uk/eppireviewer-web/ArchieCallBack"
    "CochraneOAuthRedirectUri": "https://ssru38.ioe.ac.uk/ERx/ArchieCallBack",
    "SMTP": "***REMOVED***",
    "SMTPUser": "EPPISupport@ioe.ac.uk",
    "mailFrom": "EPPISupport@ioe.ac.uk",
    "SMTPAuthentic": "***REMOVED***"
  },
  "Logging": {
    "LogLevel": {
      "Default": "Error"
    }
  }
}
